### ISSUE TITLE
Create UI-Seeds component library

### ISSUE THREAD

**Issue by ludtkemorgan** [bloom-housing/bloom#3148]
This is the first step to get the UI components in a state where it is independent of the bloom code base.

Steps:
1. Create a new repo within bloom github
2. Configure repo with proper settings
3. pull in standard configurations (tailwind, etc)
4. Pull in one UI component ?
5. publish to npm

**Comment by jaredcwhite** [comment]
Just jotting this down here, might end up as a separate issue later: we were talking about wanting to verify that styles can be exported piecemeal to downstream…aka if a consumer is using only one component, they don't end up with lots of (or all!) component CSS in the bundle.

**Comment by ludtkemorgan** [comment]
@emilyjablonski I am assigning this to you as it is related to what will be done for the stories in the audit outcome. It was put on pause as we discussed the future plan, but we know our way forward now so I think it can be unblocked. Feel free to close this issue as well if you have another ticket that better tracks it.

Here is where it stands now:
- New git repo called [ui-seeds](https://github.com/bloom-housing/ui-seeds) is created
- ExpandableText component pulled into the ui-seeds repo
- ui-seeds is published to [npm](https://www.npmjs.com/package/@bloom-housing/ui-seeds)

What is remaining:
- Pulling that component into bloom core.
- Adding auto publishing to NPM (ideally the same way ui-components is doing it now)

Options:
1. Continue with this repo and address the above. (feel free to pull in a different component as well)
2. Use the repo in name only and start from scratch
3. Take the current code and make major changes to its configuration.
